tsconfig.json

```diff
@@ -2,12 +2,14 @@
   "version": "1.8.2",
   "compilerOptions": {
     "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
Why did you introduce this or `allowSyntheticDefaultImports`?

I think the issues you're having on circ admin and registry could be because of these new rules. https://github.com/NYPL-Simplified/registry_admin/pull/13 works with version 1.3.0 but version 1.3.1 fails and I think these are significant changes when compiling and building.

Reply:
Btw, `moduleResolution` defaults to `node` when `commonjs` is being used.